snuba/settings_base.py

```diff
@@ -56,7 +56,7 @@
 DEFAULT_MAX_BATCH_SIZE = 50000
 DEFAULT_MAX_BATCH_TIME_MS = 2 * 1000
 DEFAULT_QUEUED_MAX_MESSAGE_KBYTES = 50000
-DEFAULT_QUEUED_MIN_MESSAGES = 20000
+DEFAULT_QUEUED_MIN_MESSAGES = 10000
```

Comment:
Matches the hardcoded default value from the batching kafka consumer:
https://github.com/getsentry/batching-kafka-consumer/blob/master/batching_kafka_consumer/__init__.py#L16